week1_KimChaeHyun/src/components/tab/FilterTab.module.css

```
@@ -0,0 +1,122 @@
+.tabContainer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0.5rem 1rem;
+}
+
+.tabs {
+  position: relative;
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+
+  width: 28rem;
+}
+
+.tabItem {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.3rem;
+  flex: 1;
+  color: var(--text3);
+  cursor: pointer;
+  padding: 0.5rem;
+  font-weight: 500;
+}
+
+.tabItem:hover {
+  color: var(--accent2);
+}
+
+.active {
+  color: var(--text1);
+  font-weight: bold;
+}
+
+.tabIndicator {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: calc(100% / 3 + 0.3rem);
+  height: 0.2rem;
+  background-color: var(--text1);
+  transition: transform 0.3s ease;
+}
+
+.icon {
+  font-size: 2.4rem;
+
+  &:hover{
+    color: var(--text1);
+}
+}
+
+.label {
+  font: var(--heading4);
```

```suggestion
  font: var(--heading4);
  white-space: nowrap;
```
줄바꿈 안되게 white-space 추가해주면 좀 더 자연스럽게 수정되는것 같네용